ethcore/src/miner/miner.rs

```diff
@@ -649,7 +663,12 @@ impl MinerService for Miner {
 				// Client should send message after commit to db and inserting to chain.
 				.expect("Expected in-chain blocks.");
 			let block = BlockView::new(&block);
-			block.transactions()
+			let txs = block.transactions();
+			// populate sender
+			for tx in &txs {
+				let _sender = tx.sender();
+			}
```

Comment:
I'm not sure what the value of populating the sender explicitly here rather than letting it occur lazily as it's needed. What's the purpose of this complexity here? I'd vote just dumping these 3 lines if that's ok.

The only benefit I can think of is to populate it on this thread for performance reasons, but are they really that critical?